src/VisualStudio/Setup/AssemblyRedirects.cs

```diff
@@ -3,6 +3,14 @@
 using Microsoft.VisualStudio.Shell;
 
 [assembly: ProvideBindingRedirection(
+    AssemblyName = "Microsoft.Build.Tasks.Roslyn",
```

Comment:
Is this shipping with the name Roslyn? I admit I don't have a better name, but I suspect this is something we might regret...?